cloudbuild.Dockerfile

```diff
@@ -1,5 +1,8 @@
 # The image has the environment setup for running builds and tests
 # on Google Cloud Build.
 FROM gcr.io/cloud-builders/go:debian
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends python2-dev python3-dev
+# Install bazel.
```

Comment:
Let's revert this and use the bazel container directly. FWIW, this also works, but let's keep the individual steps small.

Reply:
Reverted this now that bazel has its own step.